.github/workflows/upgrade-develop-testing.yml

```diff
@@ -63,13 +63,17 @@ jobs:
         php: [ '7.2', '8.4' ]
         db-type: [ 'mysql' ]
         db-version: [ '5.7', '8.4' ]
-        wp: [ '6.5', '6.6', '6.7' ]
+        # WordPress 4.9 is the oldest version that supports PHP 7.2.
+        wp: [ '4.9', '6.5', '6.6', '6.7' ]
         multisite: [ false, true ]
 
         exclude:
           # The PHP <= 7.3/MySQL 8.4 jobs currently fail due to mysql_native_password being disabled by default. See https://core.trac.wordpress.org/ticket/61218.
           - php: '7.2'
             db-version: '8.4'
+          # WordPress 4.9 does not support PHP 8.4.
+          - php: '8.4'
```

Comment:
According to https://make.wordpress.org/hosting/handbook/compatibility/ WP 4.9 also only supports MySQL up to 5.7.

Reply:
Correct but it does work despite not being officially supported. The PHP 8.4 test needs to be excluded because the site exits with a fatal error due to the use of `get_magic_quotes_gpc()` in 4.9.